Intersect.Server.Core/Entities/Player.Database.cs

```diff
@@ -41,7 +41,7 @@ public partial class Player
     public static bool TryFetch(LookupKey lookupKey, [NotNullWhen(true)] out Tuple<Client, Player>? tuple)
     {
         tuple = Fetch(lookupKey);
-        return tuple != default;
+        return tuple.Item1 != default || tuple.Item2 != default;
```

Comment:
Was this not working before? If it isn't I think Fetch's logic should be moved into TryFetch, and then Fetch needs to be removed

Reply:
Also this TryFetch I think should be changed to be the same thing as I suggested in User.cs